analysis/transformations_mapping_analysis.py

```diff
@@ -0,0 +1,154 @@
+"""
+# Transformations Mapping Analysis
+
+This script analyses the transformations in the
+NL-Augmenter notebook and maps each transformation into specific groups using the supplied metadata.
+@author = Saad Mahamood
+"""
+import inspect
+import os
+import pandas as pd
+from importlib import import_module
+import json
+
+from interfaces.Operation import Operation
+
+class TransformationMappingAnalysis():
+    ignore_list = ["GermanGenderSwap", "disability_transformation", "gender_neutral_rewrite",
```

Comment:
The GermanGenderSwap has been appropriately renamed now to the underscore format in https://github.com/GEM-benchmark/NL-Augmenter/pull/312.